README.md

```diff
@@ -34,3 +34,28 @@ bundle install
 bundle exec rspec
 exit
```

Comment:
I know you didn't add it but it seems superfluous (I noticed you adding it in `aliquot`).

```suggestion
```